src/operator/src/statement/ddl.rs

```diff
@@ -239,6 +240,9 @@ impl StatementExecutor {
             &create_table.table_name,
         );
 
+        // fill schema options in create table expr;
+        fill_schema_opts(create_table, &schema_opts);
```

Comment:
Question: How should we respond if the `alter database ttll` operation is performed and the tables already exist?

Reply:
> Question: How should we respond if the `alter database ttll` operation is performed and the tables already exist?

This PR does not break current behavior, it just fix the problem that database ttl is not applied to regions (neither in memory and region options in metadata).

Altering database options is another story and won't be covered in this PR.